### PR TITLE
Agent install config improvements

### PR DESCRIPTION
**Pull request by zaneb**
I somehow missed this file in my previous round of reviews.
These changes should be backported to 4.12.

### CODE REVIEW

**Comment by Amrita42** [comment]
ok noted . will implement on monday.

File: modules/installation-bare-metal-agent-installer-config-yaml.adoc [hunk 1]
```diff
@@ -14,68 +14,52 @@ You can customize the `install-config.yaml` file to specify more details about y
 apiVersion: v1
 baseDomain: example.com <1>
 compute: <2>
-- hyperthreading: Enabled <3>
-  name: worker
-  replicas: 0 <4>
+- name: worker
```
Comment (Amrita42) [inline modules/installation-bare-metal-agent-installer-config-yaml.adoc:17]: @zaneb  should there be a hypen for _name_ ?

File: modules/installation-bare-metal-agent-installer-config-yaml.adoc [hunk 2]
```diff
@@ -14,68 +14,52 @@ You can customize the `install-config.yaml` file to specify more details about y
...
-<11> You must set the platform to `none` for a single-node cluster. You can also set the platform to `vSphere` and `baremetal`.
+<7> The subnet prefix length to assign to each individual node. For example, if `hostPrefix` is set to `23`, then each node is assigned a `/23` subnet out of the given `cidr`, which allows for 510 (2^(32 - 23) - 2) pod IP addresses. If you are required to provide access to nodes from an external network, configure load balancers and routers to manage the traffic.
+<8> The cluster network plugin to install. The supported values are `OVNKubernetes` (default value) and `OpenShiftSDN`.
+<9> The IP address pool to use for service IP addresses. You can enter only one IP address pool. This block must not overlap with existing physical networks. If you need to access the services from an external network, configure load balancers and routers to manage the traffic.
+<10> You must set the platform to `none` for a single-node cluster. For multi-node clusters, you can set the platform to `vsphere` or `baremetal`.
```
Comment (Amrita42) [inline modules/installation-bare-metal-agent-installer-config-yaml.adoc:58]: re-phrased

File: modules/installation-bare-metal-agent-installer-config-yaml.adoc (hunk 1 above)
Comment (Amrita42) [reply modules/installation-bare-metal-agent-installer-config-yaml.adoc:17]: yes it should be , ignore

File: modules/installation-bare-metal-agent-installer-config-yaml.adoc (hunk 2 above)
Comment (Amrita42) [reply modules/installation-bare-metal-agent-installer-config-yaml.adoc:58]: @zaneb Opened https://github.com/openshift/openshift-docs/pull/55912 ptal.

**Comment by zaneb** [comment]
/close
In favour of #55912

**Pull request closed without merging**